docs/content/direct/release-notes.md

```diff
@@ -4,14 +4,16 @@ The following sections list the known issues for each release. The issue list is
 
 ## 0.25.0 and its candidates
 
-The main advance in this release is finishing the implementation of the create-only feature. It is now available for use.
+* The main advance in this release is finishing the implementation of the create-only feature. It is now available for use.
+* Transport controller's `max-num-wrapped` is defaulted to 1 in the core Helm chart.
```

Comment:
I would call this out as a _change_ in the default, from a huge number down to 1.